.github/workflows/migrations.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 
 env:
-  TARGET_VERSION: 1.4.1
+  TARGET_VERSION: 1.4.2
```

Comment:
It's maybe useful to add a comment here, given our current internal discussions.. Something that says that the DB was created with this version of fractal-server, and is now used while upgrading to the current version.